perl-lib/OESS/lib/OESS/Interface_test.t

```diff
@@ -0,0 +1,14 @@
+use OESS::Interface;
+use Test::More tests => 1;
+use Log::Log4perl;
+use OESS::Database;
+use OESS::DB::Interface;
+
+$OESS::Database::ENABLE_DEVEL=1;
+Log::Log4perl->init("/etc/oess/logging.conf");
+my $db = OESS::Database->new();
+my $interface = OESS::Interface->new(	db 	=> $db,
+					name	=> "asd",
+					node	=> 
+						);
+ok(defined($interface),"Object interface initiated");
```

Comment:
There should be way more tests we can do on the interface object itself.  For example fetching the name, checking to see if it has a cloud_interconnect_id, getting a list of ACL, checking to see if the VLAN is in use.

Reply:
On it!